src/Arcus.BackgroundJobs.Tests.Unit/Databricks/SpyLogger.cs

```diff
@@ -0,0 +1,57 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Collections.ObjectModel;
+using System.Linq;
+using GuardNet;
+using Microsoft.Extensions.Logging;
+
+namespace Arcus.BackgroundJobs.Tests.Unit.Databricks
+{
+    /// <summary>
+    /// Spy (stub) <see cref="ILogger{TCategoryName}"/> implementation to track the logged messages.
+    /// </summary>
+    /// <typeparam name="T">The type who's name is used for the logger category name.</typeparam>
+    public class SpyLogger<T> : ILogger<T>
```

Comment:
Something for Arcus testing?

Reply:
Yes! I like that. It's now used in two different projects with two different interface. It's definitely something we could need somewhere else.
I'll make an issue but leave the PR as it is for now. Ok?

Reply:
Jup, fine for me!

Reply:
#42